ksrates/plot_paralogs.py

```diff
@@ -7,7 +7,7 @@
 import ksrates.fc_extract_ks_list as fc_extract_ks_list
 import ksrates.fc_check_input as fcCheck
 import ksrates.fc_configfile as fcConf
-
+from ksrates.fc_rrt_correction import _ADJUSTMENT_TABLE
```

Comment:
Since _MIXED_ADJUSTED_PLOT_FILENAME is a variable coming from a module that is imported in the first lines, I call it as `fcPlot._MIXED_ADJUSTED_PLOT_FILENAME` directly in the code (line 131).
Instead, since _ADJUSTMENT_TABLE comes from a module that is not imported in the first lines because not needed, I import only that variable from it at the beginning of the script.

Reply:
Alright.